DocSum/docker/xeon/docker_compose.yaml

```diff
@@ -68,6 +68,18 @@ services:
       - DOC_BASE_URL=${BACKEND_SERVICE_ENDPOINT}
     ipc: host
     restart: always
+  docsum-xeon-react-ui-server:
+    image: opea/docsum-react-ui:latest
+    container_name: docsum-xeon-react-ui-server
+    depends_on:
+      - docsum-xeon-backend-server
+    build:
+      args:
+        - BACKEND_SERVICE_ENDPOINT=${BACKEND_SERVICE_ENDPOINT}
+    ports:
+      - "5174:80"
```

Comment:
In the docker_compose.yaml of Gaudi, you introduced the environment parameter, but Xeon should be the same; why is it not included here?

Reply:
removed the environment section in gaudi as it is not needed